packages/resolvers/default/src/DefaultResolver.js

```diff
@@ -179,6 +179,33 @@ class NodeResolver {
       return builtin;
     }
 
+    // Resolve peer dependencies
+    // Finds the first package that does not specify filename as a peer dependency
+    // and resolve from there. If there was no package, or all packages specified
+    // filename as a peer dependency, the module is marked as external
+    let pkg = await this.findPackage(dir);
+    let externalPeerDependency = false;
+    while (pkg) {
+      let isPeerDependency = false;
+      let hasPeerDependencies = pkg.peerDependencies != undefined;
+      if (hasPeerDependencies) {
+        isPeerDependency = Object.keys(pkg.peerDependencies).includes(filename);
```

Comment:
```suggestion
        isPeerDependency = filename in pkg.peerDependencies
```
would probably be faster?

And `hasPeerDependencies` can be inlined above.
`externalPeerDependency` is always  set to `isPeerDependency`, so `isPeerDependency` isn't really need either.

Reply:
Yes, that would indeed be faster :-) will update ...